test/primus.test.js

```diff
@@ -680,7 +686,8 @@ describe('Primus', function () {
 
     it('applies the options to the Primus server', function (done) {
       var port = common.port
-        , primus = Primus.createServer({ port: port, transformer: 'engine.io' });
+        , primus = Primus.createServer(
```

Comment:
Minor style nit: get rid of the `port` variable and use one line per option in the options object.